example/src/screens/timelineCalendar.tsx

```diff
@@ -158,6 +159,7 @@ export default class TimelineCalendarScreen extends Component {
           markedDates={this.marked}
         />
         <Timeline
+          date={currentDate}
```

Comment:
Since this is a new prop, the existing usages do not pass it to the timeline component, thus a new event is not created for them. Maybe we should mention it in the prop/feature description (i.e. 'onBackgroundLongPress' / 'date' props)?

Reply:
Is there a way to determine internally on which date the user long-pressed without passing the date from the screen?

Reply:
I can default it to a `new Date()` but that won't be necessarily true when we'll have 3 days view. 
Anyway, I will update the prop description to make it more clear

Reply:
Done